ts/src/coinbase.ts

```diff
@@ -1249,10 +1255,35 @@ export default class coinbase extends Exchange {
         const expires = this.safeInteger (options, 'expires', 1000);
         const now = this.milliseconds ();
         if ((timestamp === undefined) || ((now - timestamp) > expires)) {
-            const currencies = await this.v2PublicGetCurrencies (params);
+            const fiatResponse = await this.v2PublicGetCurrencies (params);
+            //
+            //    [
+            //        "data": {
+            //            id: 'IMP',
+            //            name: 'Isle of Man Pound',
+            //            min_size: '0.01'
+            //        },
+            //        ...
+            //    ]
+            //
+            const cryptoResponse = await this.v2PublicGetCurrenciesCrypto (params);
```

Comment:
we should do this in "parallel" with promise.all

Reply:
Let me make changes to my PRs. It's helps me remember to make these changes in the future. This is a non-critical PR, it can be merged at any time and is not urgent.